prisma/schema.prisma

```diff
@@ -0,0 +1,26 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider  = "postgresql"
+  url       = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
+}
+
+model User {
+  address        String        @id @unique
+  last_connected DateTime?     @db.Timestamptz(6)
+  Transaction    Transaction[]
+}
+
+model Transaction {
+  transaction_hash String    @id
```

Comment:
any chance of collision across chains if this is the `id`? might need a composite key

Reply:
I don't think it's super likely, but yeah we can do that